README.md

```diff
@@ -0,0 +1,41 @@
+# Picover
```

Comment:
First and simple iteration for project readme, added two ideas that already came to my mind for the next iterations:
- adding badges
- adding linked contributors with names and avatars